roles/nsx_node/tasks/main.yml

```diff
@@ -0,0 +1,114 @@
+---
+- name: NSX | Download CNI rpm
+  get_url:
+    url: "{{ nsx_cni_url }}"
+    dest: /tmp/cni.rpm
+  tags: nsx_node
```

Comment:
Same, not sure how tags are useful here

Reply:
OK I am removing the tags only on the two points you commented.
I have put tags everywhere because of this: https://github.com/openshift/openshift-ansible/blob/master/docs/best_practices_guide.adoc#All-tasks-in-a-role-SHOULD-be-tagged-with-the-role-name

Anyway, I will commit in a moment.

Reply:
fixed with 11e4f05048c54c2d7b2d1f981cb101a0471be058
Let me know if you want me to remove the tags everywhere.